### DISCUSSION TITLE
can't find or create a config file.

### DISCUSSION THREAD

**Discussion by codenoobforreal** [ghostty-org/ghostty#4231] [Q&A] [answered]
I tried all the locations of the config file described in the doc then found nothing.
So I ran `sudo vi ~/.config/ghostty/config` and tried to create one and got `E212: Can't open file for writing`.

Mac

**Comment by rhodes-b** [comment, marked as answer]
the file is not created for you in the 1.0 release #3460 changes that and is in the 1.0.1 release

for now you can `mkdir -p ~/.config/ghostty && touch ~/.config/ghostty/config`

**Reply by codenoobforreal** [reply]
thanks.
